src/Microsoft.Data.SqlClient/tests/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj

```diff
@@ -45,11 +45,7 @@
     <Compile Include="SqlConnectionTest.RetrieveStatistics.cs" />
     <Compile Include="FakeDiagnosticListenerObserver.cs" />
     <Compile Include="SqlMetaDataTest.cs" />
-    <Compile Include="ExceptionTest.cs" />
-    <Compile Include="TcpDefaultForAzureTest.cs" />
     <Compile Include="SqlConnectionStringBuilderTest.cs" />
-    <!-- <Compile Include="DiagnosticTest.cs" /> -->
```

Comment:
Please keep this line. I'd like to re-enable this test some day when I can get RemoteExecutor working..